src/kinetics/Falloff.cpp

```diff
@@ -30,8 +30,17 @@ void Troe::init(const vector_fp& c)
             c.size());
     }
     m_a = c[0];
-    m_rt3 = 1.0/c[1];
-    m_rt1 = 1.0/c[2];
+    
+    if (std::abs(c[1]) < SmallNumber)
+        m_rt3 = 1.0/SmallNumber;
```

Comment:
* I would suggest setting the value to `std::numeric_limits<double>::infinity()`, so that the inverse operation in `Troe::getParameters` provides the correct output.
* Please do not use `if/else` constructs without braces.